content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -2,89 +2,130 @@
-(updates go through both directions).
+Synchronize Google Calendar with Odoo to see and manage meetings from both platforms (updates go
+in both directions). This integration helps organize your schedule so you never miss a meeting.
 
 Setup in Google
 ===============
-- Go to `Google APIs platform <https://console.developers.google.com>`__ 
-  to generate Google Calendar API credentials. Log in with your Google account. 
 
-- Go to the API & Services page.
+Enable Google Calendar API
+--------------------------
 
-.. image:: media/google_calendar_credentials00.png
-    :align: center
+To begin, you need to create a new Google API project and enable the Google Calendar API. First, go
+to the `Google API Console <https://console.developers.google.com_>`_ and log into your Google
+account.
 
-- Search for *Google Calendar API* and select it.
+.. note::
+   If this is your first time visiting this page, Google will prompt you to enter your country and
+   agree to the Terms of Service. Select your country from the drop-down list and agree to the
+   :abbr:`ToS (Terms of Service)`.
+  
+Next, click :guilabel:`Select a project` and select or create an API project to store credentials.
 
-.. image:: media/google_calendar_credentials01.png
-    :align: center
+.. image:: google_calendar_credentials/new_api_project.png
+   :align: center
+   :alt: Create a new API project to store credentials.
 
-.. image:: media/google_calendar_credentials02.png
-    :align: center
+.. tip::
+   Give the API Project a clear name like "Odoo Sync" so you can easily find it.
 
-- Enable the API.
+Then, open the API Project and click :guilabel:`Enable APIs and Services`.
 
-.. image:: media/google_calendar_credentials03.png
-    :align: center
+.. image:: google_calendar_credentials/enable_apis_services.png
+   :align: center
+   :alt: Enable APIs and Services on the API Project.
 
-- Select or create an API project to store the credentials if not yet done 
-  before. Give it an explicit name (e.g. Odoo Sync).
+After that, search for *Google Calendar API* using the search bar and select **Google Calendar
+API** from the search results. Click :guilabel:`Enable`.
 
-- Create credentials.
+.. image:: google_calendar_credentials/enable_google_cal_api.png
+   :align: center
+   :alt: Enable the Google Calendar API.
 
-.. image:: media/google_calendar_credentials04.png
-    :align: center
+Create credentials
+------------------
 
-- Select *Web browser (Javascript)* 
-  as calling source and *User data* as kind of data.
+Now that you have created your API project and enabled the Google Calendar API, you need to create
+credentials. Begin by clicking :guilabel:`Create Credentials`. Google will then guide you through
+four steps to create your API credentials.
 
-.. image:: media/google_calendar_credentials05.png
-    :align: center
+- In the first step, **Credential Type**, select the *Google Calendar API* and *User Data* options.
```

Comment:
These all need :guilabel: since they're fields or on-screen selection items

```suggestion
- In the first step, :guilabel:`Credential Type`, select the :guilabel:`Google Calendar API` and :guilabel:`User Data` options.
```